src/utils.js

```diff
@@ -215,7 +215,7 @@ function getNotificationsStatus(
     default:
       query = {
         query: `{
-          notifications(${parameters}) {
+          notifications(first: 1000 ${parameters}) {
```

Comment:
Parameters need to be separated with commas right?

Reply:
no that's not the issue, it's because this query input a parameters first and limit and somehow when adding first it returns more than expected